src/sql/workbench/contrib/resourceViewer/browser/resourceViewerTable.ts

```diff
@@ -0,0 +1,94 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Table } from 'sql/base/browser/ui/table/table';
+import { attachTableStyler } from 'sql/platform/theme/common/styler';
+import { RowSelectionModel } from 'sql/base/browser/ui/table/plugins/rowSelectionModel.plugin';
+
+import { IWorkbenchThemeService } from 'vs/workbench/services/themes/common/workbenchThemeService';
+import { Dimension } from 'vs/base/browser/dom';
+import { textFormatter, slickGridDataItemColumnValueExtractor } from 'sql/base/browser/ui/table/formatters';
+import { TableDataView } from 'sql/base/browser/ui/table/tableDataView';
+
+export interface ResourceViewerTableViewState {
+	scrollTop: number;
+	scrollLeft: number;
+}
+
+export class ResourceViewerTable {
+
+	private _resourceViewerTable!: Table<Slick.SlickData>;
+	private _data: TableDataView<Slick.SlickData> | undefined;
+	private _overlay: HTMLElement;
+
+	constructor(parent: HTMLElement, @IWorkbenchThemeService private _themeService: IWorkbenchThemeService) {
+		this._overlay = document.createElement('div');
+		this._overlay.className = 'overlayWidgets';
+		this._overlay.style.width = '100%';
+		this._overlay.style.zIndex = '4';
```

Comment:
any specific reason you are modifying the z-index? z-index issues can be annoying to debug >.>

Reply:
Just removed this - it was for the original overlay the profiler uses for stuff like the find widget but I don't need that right now